__tests__/commands/install/integration-deduping.js

```diff
@@ -1,8 +1,10 @@
 /* @flow */
 
 import {getPackageVersion, runInstall} from '../_helpers.js';
+import * as fs from '../../../src/util/fs.js';
```

Comment:
You mean I could only `import stat` instead of `* as fs`?
Not sure if that would be better when the number of imports  from fs grow.
Everything grouped into one import makes sense rather than a lot functions that need sorting and formatting.
What should be the guideline here?